packages/web-app-files/src/views/spaces/Projects.vue

```diff
@@ -6,20 +6,21 @@
       ref="createNewSpaceButton"
       key="new-space-menu-btn-enabled"
       :aria-label="$gettext('Create a new space')"
-      variation="primary"
-      appearance="filled"
-      class="oc-mb-l"
+      variation="inverse"
```

Comment:
This should be fixed in the design system. The primary variation is supposed to look like you styled it manually now. Please revert the changes to the button here.